flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/TableSchema.java

```diff
@@ -43,7 +44,28 @@
 import static org.apache.flink.table.types.utils.TypeConversions.fromLegacyInfoToDataType;
 
 /**
- * A table schema that represents a table's structure with field names and data types.
+ * <p>A table schema that represents a table's structure with field names and data types and some
+ * constraint information (e.g. primary key, unique key).</p><br/>
+ *
+ * <p>Concepts about primary key and unique key:</p>
```

Comment:
do we distinguish primary key from unique key? in current javadoc, they have no difference.

Reply:
The difference between primary key and unique key is that there is only one primary key and there can be more than one unique key. And a primary key doesn't need to be declared in unique key list again.

I will add this to the class Javadoc.